index.js

```diff
@@ -1,7 +1,82 @@
+console.log("I'm ready!");
 // Iteration 1: Names and Input
 
+let hacker1 = prompt("Driver name"), hacker2 = prompt("Navigator name");
```

Comment:
Esto esta genial!, pero solo funcionaria en un navegador que es el que tiene la funcion prompt, en una terminal, normalita no esta, por ello os pongo como seria sin, pero vamos que es lo mismo, le poneis valores fijos y listo!
```suggestion
const hacker1 = 'Alejandro';
const hacker2 = 'Naiim';
```
Y acordaros si no reasignais el valor, siempre siempre siempre const :D